src/libraries/System.Security.Cryptography/tests/Rfc2898OneShotTests.cs

```diff
@@ -289,7 +289,7 @@ public static void Pbkdf2_Rfc6070_HighIterations()
             string password = "password";
             int iterations = 16777216;
             byte[] expected = "eefe3d61cd4da4e4e9945b3d6ba2158c2634e984".HexToByteArray();
-            byte[] salt = Encoding.UTF8.GetBytes("salt");
+            ReadOnlySpan<byte> salt = "salt"u8;
```

Comment:
Nit: This is now unit testing a different overload. Was this intentional?

Reply:
I would say that for that particular test, which overload is called does not matter. The overload correctness test is `Pbkdf2_PasswordBytes_Compare`.

Reply:
Yeah, it didn't seem like it mattered in this case.  Thanks for calling it out, though.